app/javascript/gobierto_data/webapp/pages/Home.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <div class="gobierto-data">
+    <div class="pure-g gutters m_b_1">
+      <div class="pure-u-1 pure-u-lg-4-4">
+        <Nav
+          :active-tab="activeTabIndex"
+          @active-tab="activeTabIndex = $event"
+        />
+        <DataSets />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import Nav from "./../components/Nav.vue";
+import DataSets from "./../pages/DataSets.vue";
```

Comment:
Module not found: Error: [CaseSensitivePathsPlugin] `gobierto/app/javascript/gobierto_data/webapp/pages/DataSets.vue` does not match the corresponding path on disk `Datasets.vue`.